### ISSUE TITLE
Whirpool GUI stuck on CLI download

### ISSUE THREAD

**Issue by meeDamian** [Samourai-Wallet/whirlpool-gui#11]
`whirpool-gui.app` `v0.7.0` on Mac OS Catalina 10.15 gets stuck on downloading the CLI indefinitely.

<img width="1392" alt="MacOS 2019-08-09 at 09 29 29" src="https://user-images.githubusercontent.com/526657/62750011-c7af8c00-ba88-11e9-8aac-51238353705d.png">

`~/Library/Application\ Support/whirlpool-gui/` contains two copies of the downloaded CLI:

```bash
$ ls -lh ~/Library/Application\ Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run*
-rw-r--r--@ 1 mee  staff    40M Aug  9 09:21 /Users/mee/Library/Application Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run (1).jar
-rw-r--r--@ 1 mee  staff    40M Aug  9 09:21 /Users/mee/Library/Application Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run.jar
```

Relevant logs:

```log
[2019-08-09 09:36:19.932] [info] CLI reloading...
[2019-08-09 09:36:20.570] [info] using CLI_API 0.7 { CLI_VERSION: '0.7.2',
  CLI_CHECKSUM:
   '8062e79a26d85372fc8d954c1ff321e598d163390fe3b82bfcb87adb2de54ebb' }
[2019-08-09 09:36:20.572] [error] CLI not found: /Users/mee/Library/Application Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run.jar
[2019-08-09 09:36:20.573] [info] CLI downloading: https://github.com/Samourai-Wallet/whirlpool-runtimes/releases/download/cli-0.7.2/whirlpool-client-cli-0.7.2-run.jar
[2019-08-09 09:36:20.942] [info] cliService is not configured.
[2019-08-09 09:36:20.948] [info] cliService.setCliLocal: true
[2019-08-09 09:36:21.004] [info] CLI reloading...
[2019-08-09 09:36:21.202] [info] using CLI_API 0.7 { CLI_VERSION: '0.7.2',
  CLI_CHECKSUM:
   '8062e79a26d85372fc8d954c1ff321e598d163390fe3b82bfcb87adb2de54ebb' }
[2019-08-09 09:36:21.203] [error] CLI not found: /Users/mee/Library/Application Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run.jar
[2019-08-09 09:36:21.203] [info] CLI downloading: https://github.com/Samourai-Wallet/whirlpool-runtimes/releases/download/cli-0.7.2/whirlpool-client-cli-0.7.2-run.jar
[2019-08-09 09:36:22.386] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.388] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.392] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.393] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.577] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.578] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.581] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:22.582] [verbose] CLI downloading, progress=0
[2019-08-09 09:36:23.154] [verbose] CLI downloading, progress=0.001238493168946908
[2019-08-09 09:36:23.155] [verbose] CLI downloading, progress=0.001238493168946908
[2019-08-09 09:36:23.554] [verbose] CLI downloading, progress=0.003413436468292043
[2019-08-09 09:36:23.557] [verbose] CLI downloading, progress=0.003413436468292043
[2019-08-09 09:36:23.655] [verbose] CLI downloading, progress=0.00714513640800491
[2019-08-09 09:36:23.656] [verbose] CLI downloading, progress=0.00714513640800491
[2019-08-09 09:36:24.055] [verbose] CLI downloading, progress=0.012950002980891592
[2019-08-09 09:36:24.056] [verbose] CLI downloading, progress=0.012950002980891592
[2019-08-09 09:36:24.156] [verbose] CLI downloading, progress=0.023315836146760666
[2019-08-09 09:36:24.157] [verbose] CLI downloading, progress=0.023315836146760666
[2019-08-09 09:36:24.556] [verbose] CLI downloading, progress=0.027047536086473534
[2019-08-09 09:36:24.558] [verbose] CLI downloading, progress=0.027047536086473534
[2019-08-09 09:36:24.657] [verbose] CLI downloading, progress=0.027047536086473534
[2019-08-09 09:36:24.658] [verbose] CLI downloading, progress=0.027047536086473534
[2019-08-09 09:36:25.056] [verbose] CLI downloading, progress=0.03202313600609069
[2019-08-09 09:36:25.058] [verbose] CLI downloading, progress=0.03202313600609069
[2019-08-09 09:36:25.158] [verbose] CLI downloading, progress=0.059105590087557704
[2019-08-09 09:36:25.160] [verbose] CLI downloading, progress=0.059105590087557704
[2019-08-09 09:36:25.557] [verbose] CLI downloading, progress=0.0828091445989186
[2019-08-09 09:36:25.559] [verbose] CLI downloading, progress=0.0828091445989186
[2019-08-09 09:36:25.659] [verbose] CLI downloading, progress=0.09510208904646078
[2019-08-09 09:36:25.660] [verbose] CLI downloading, progress=0.09510208904646078
[2019-08-09 09:36:26.058] [verbose] CLI downloading, progress=0.10739503349400295
[2019-08-09 09:36:26.059] [verbose] CLI downloading, progress=0.10739503349400295
[2019-08-09 09:36:26.159] [verbose] CLI downloading, progress=0.11441826656068682
[2019-08-09 09:36:26.162] [verbose] CLI downloading, progress=0.11441826656068682
[2019-08-09 09:36:26.558] [verbose] CLI downloading, progress=0.12983315605583193
[2019-08-09 09:36:26.560] [verbose] CLI downloading, progress=0.12983315605583193
[2019-08-09 09:36:26.663] [verbose] CLI downloading, progress=0.14115192178902658
[2019-08-09 09:36:26.664] [verbose] CLI downloading, progress=0.14115192178902658
[2019-08-09 09:36:27.058] [verbose] CLI downloading, progress=0.1600784278175136
[2019-08-09 09:36:27.060] [verbose] CLI downloading, progress=0.1600784278175136
[2019-08-09 09:36:27.162] [verbose] CLI downloading, progress=0.17178686503644727
[2019-08-09 09:36:27.163] [verbose] CLI downloading, progress=0.17178686503644727
[2019-08-09 09:36:27.563] [verbose] CLI downloading, progress=0.18934952086484777
[2019-08-09 09:36:27.565] [verbose] CLI downloading, progress=0.18934952086484777
[2019-08-09 09:36:27.665] [verbose] CLI downloading, progress=0.19929986323626503
[2019-08-09 09:36:27.667] [verbose] CLI downloading, progress=0.19929986323626503
[2019-08-09 09:36:28.060] [verbose] CLI downloading, progress=0.21490958847427966
[2019-08-09 09:36:28.061] [verbose] CLI downloading, progress=0.21490958847427966
[2019-08-09 09:36:28.165] [verbose] CLI downloading, progress=0.2236909163884799
[2019-08-09 09:36:28.166] [verbose] CLI downloading, progress=0.2236909163884799
[2019-08-09 09:36:28.561] [verbose] CLI downloading, progress=0.2418380794454889
[2019-08-09 09:36:28.562] [verbose] CLI downloading, progress=0.2418380794454889
[2019-08-09 09:36:28.667] [verbose] CLI downloading, progress=0.2541310238930311
[2019-08-09 09:36:28.668] [verbose] CLI downloading, progress=0.2541310238930311
[2019-08-09 09:36:29.066] [verbose] CLI downloading, progress=0.275789803483382
[2019-08-09 09:36:29.067] [verbose] CLI downloading, progress=0.275789803483382
[2019-08-09 09:36:29.167] [verbose] CLI downloading, progress=0.2882775836737937
[2019-08-09 09:36:29.169] [verbose] CLI downloading, progress=0.2882775836737937
[2019-08-09 09:36:29.566] [verbose] CLI downloading, progress=0.30700925395941125
[2019-08-09 09:36:29.567] [verbose] CLI downloading, progress=0.30700925395941125
[2019-08-09 09:36:29.667] [verbose] CLI downloading, progress=0.317544103559437
[2019-08-09 09:36:29.669] [verbose] CLI downloading, progress=0.317544103559437
[2019-08-09 09:36:30.066] [verbose] CLI downloading, progress=0.33588610235931554
[2019-08-09 09:36:30.067] [verbose] CLI downloading, progress=0.33588610235931554
[2019-08-09 09:36:30.168] [verbose] CLI downloading, progress=0.34544677324499384
[2019-08-09 09:36:30.169] [verbose] CLI downloading, progress=0.34544677324499384
[2019-08-09 09:36:30.567] [verbose] CLI downloading, progress=0.3647629507592199
[2019-08-09 09:36:30.568] [verbose] CLI downloading, progress=0.3647629507592199
[2019-08-09 09:36:30.668] [verbose] CLI downloading, progress=0.377055895206762
[2019-08-09 09:36:30.669] [verbose] CLI downloading, progress=0.377055895206762
[2019-08-09 09:36:31.068] [verbose] CLI downloading, progress=0.39890951053998247
[2019-08-09 09:36:31.069] [verbose] CLI downloading, progress=0.39890951053998247
[2019-08-09 09:36:31.169] [verbose] CLI downloading, progress=0.4117869622161332
[2019-08-09 09:36:31.170] [verbose] CLI downloading, progress=0.4117869622161332
[2019-08-09 09:36:31.569] [verbose] CLI downloading, progress=0.43129797547322873
[2019-08-09 09:36:31.570] [verbose] CLI downloading, progress=0.43129797547322873
[2019-08-09 09:36:31.669] [verbose] CLI downloading, progress=0.4406638106160375
[2019-08-09 09:36:31.671] [verbose] CLI downloading, progress=0.4406638106160375
[2019-08-09 09:36:32.069] [verbose] CLI downloading, progress=0.45939548090165505
[2019-08-09 09:36:32.070] [verbose] CLI downloading, progress=0.45939548090165505
[2019-08-09 09:36:32.171] [verbose] CLI downloading, progress=0.46993490366337165
[2019-08-09 09:36:32.173] [verbose] CLI downloading, progress=0.46993490366337165
[2019-08-09 09:36:32.569] [verbose] CLI downloading, progress=0.4898355884062062
[2019-08-09 09:36:32.570] [verbose] CLI downloading, progress=0.4898355884062062
[2019-08-09 09:36:32.673] [verbose] CLI downloading, progress=0.5021239596920576
[2019-08-09 09:36:32.674] [verbose] CLI downloading, progress=0.5021239596920576
[2019-08-09 09:36:33.070] [verbose] CLI downloading, progress=0.5237827392824085
[2019-08-09 09:36:33.071] [verbose] CLI downloading, progress=0.5237827392824085
[2019-08-09 09:36:33.173] [verbose] CLI downloading, progress=0.53666476412025
[2019-08-09 09:36:33.174] [verbose] CLI downloading, progress=0.53666476412025
[2019-08-09 09:36:33.570] [verbose] CLI downloading, progress=0.5559809416344761
[2019-08-09 09:36:33.571] [verbose] CLI downloading, progress=0.5559809416344761
[2019-08-09 09:36:33.674] [verbose] CLI downloading, progress=0.5653467767772848
[2019-08-09 09:36:33.675] [verbose] CLI downloading, progress=0.5653467767772848
[2019-08-09 09:36:34.070] [verbose] CLI downloading, progress=0.5846675274532017
[2019-08-09 09:36:34.072] [verbose] CLI downloading, progress=0.5846675274532017
[2019-08-09 09:36:34.174] [verbose] CLI downloading, progress=0.595786884281836
[2019-08-09 09:36:34.175] [verbose] CLI downloading, progress=0.595786884281836
[2019-08-09 09:36:34.571] [verbose] CLI downloading, progress=0.6162720762532792
[2019-08-09 09:36:34.572] [verbose] CLI downloading, progress=0.6162720762532792
[2019-08-09 09:36:34.674] [verbose] CLI downloading, progress=0.6291495279294298
[2019-08-09 09:36:34.676] [verbose] CLI downloading, progress=0.6291495279294298
[2019-08-09 09:36:35.072] [verbose] CLI downloading, progress=0.6502238002911722
[2019-08-09 09:36:35.075] [verbose] CLI downloading, progress=0.6502238002911722
[2019-08-09 09:36:35.175] [verbose] CLI downloading, progress=0.663101251967323
[2019-08-09 09:36:35.177] [verbose] CLI downloading, progress=0.663101251967323
[2019-08-09 09:36:35.572] [verbose] CLI downloading, progress=0.682417429481549
[2019-08-09 09:36:35.574] [verbose] CLI downloading, progress=0.682417429481549
[2019-08-09 09:36:35.675] [verbose] CLI downloading, progress=0.6929568522432656
[2019-08-09 09:36:35.677] [verbose] CLI downloading, progress=0.6929568522432656
[2019-08-09 09:36:36.073] [verbose] CLI downloading, progress=0.7122730297574916
[2019-08-09 09:36:36.074] [verbose] CLI downloading, progress=0.7122730297574916
[2019-08-09 09:36:36.176] [verbose] CLI downloading, progress=0.7239814669764253
[2019-08-09 09:36:36.178] [verbose] CLI downloading, progress=0.7239814669764253
[2019-08-09 09:36:36.573] [verbose] CLI downloading, progress=0.7450557393381677
[2019-08-09 09:36:36.574] [verbose] CLI downloading, progress=0.7450557393381677
[2019-08-09 09:36:36.677] [verbose] CLI downloading, progress=0.7591022054715355
[2019-08-09 09:36:36.679] [verbose] CLI downloading, progress=0.7591022054715355
[2019-08-09 09:36:37.073] [verbose] CLI downloading, progress=0.7799816420904084
[2019-08-09 09:36:37.075] [verbose] CLI downloading, progress=0.7799816420904084
[2019-08-09 09:36:37.178] [verbose] CLI downloading, progress=0.7934436009951676
[2019-08-09 09:36:37.180] [verbose] CLI downloading, progress=0.7934436009951676
[2019-08-09 09:36:37.574] [verbose] CLI downloading, progress=0.8063210526713184
[2019-08-09 09:36:37.576] [verbose] CLI downloading, progress=0.8063210526713184
[2019-08-09 09:36:37.679] [verbose] CLI downloading, progress=0.8069101330616176
[2019-08-09 09:36:37.680] [verbose] CLI downloading, progress=0.8069101330616176
[2019-08-09 09:36:37.769] [verbose] CLI downloading, progress=0.8152012871163579
[2019-08-09 09:36:37.770] [verbose] CLI downloading, progress=0.8152012871163579
[2019-08-09 09:36:37.772] [verbose] CLI downloading, progress=0.8152012871163579
[2019-08-09 09:36:37.773] [verbose] CLI downloading, progress=0.8152012871163579
[2019-08-09 09:36:37.775] [verbose] CLI downloading, progress=0.8152012871163579
[2019-08-09 09:36:37.776] [verbose] CLI downloading, progress=0.8152012871163579
[2019-08-09 09:36:37.806] [info] CLI: download success
[2019-08-09 09:36:37.807] [info] CLI: download success
[2019-08-09 09:36:37.922] [info] using CLI_API 0.7 { CLI_VERSION: '0.7.2',
  CLI_CHECKSUM:
   '8062e79a26d85372fc8d954c1ff321e598d163390fe3b82bfcb87adb2de54ebb' }
[2019-08-09 09:36:37.927] [info] using CLI_API 0.7 { CLI_VERSION: '0.7.2',
  CLI_CHECKSUM:
   '8062e79a26d85372fc8d954c1ff321e598d163390fe3b82bfcb87adb2de54ebb' }
[2019-08-09 09:36:38.180] [verbose] CLI downloading, progress=0.8280787387925086
[2019-08-09 09:36:38.181] [verbose] CLI downloading, progress=0.8280787387925086
[2019-08-09 09:36:38.402] [error] CLI is invalid: /Users/mee/Library/Application Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run.jar, 5062dbc03a8767c1cabf7af37c2849acfa9cbe7d62660fb54c5dd263c42e4158 vs 8062e79a26d85372fc8d954c1ff321e598d163390fe3b82bfcb87adb2de54ebb
[2019-08-09 09:36:38.403] [error] CLI: download failed
[2019-08-09 09:36:38.408] [error] CLI is invalid: /Users/mee/Library/Application Support/whirlpool-gui/whirlpool-client-cli-0.7.2-run.jar, 5062dbc03a8767c1cabf7af37c2849acfa9cbe7d62660fb54c5dd263c42e4158 vs 8062e79a26d85372fc8d954c1ff321e598d163390fe3b82bfcb87adb2de54ebb
[2019-08-09 09:36:38.409] [error] CLI: download failed
[2019-08-09 09:36:38.679] [verbose] CLI downloading, progress=0.8397826028497514
[2019-08-09 09:36:38.680] [verbose] CLI downloading, progress=0.8397826028497514
[2019-08-09 09:36:39.179] [verbose] CLI downloading, progress=0.8491484379925602
[2019-08-09 09:36:39.181] [verbose] CLI downloading, progress=0.8491484379925602
[2019-08-09 09:36:39.681] [verbose] CLI downloading, progress=0.858514273135369
[2019-08-09 09:36:39.682] [verbose] CLI downloading, progress=0.858514273135369
[2019-08-09 09:36:40.182] [verbose] CLI downloading, progress=0.871980805201819
[2019-08-09 09:36:40.183] [verbose] CLI downloading, progress=0.871980805201819
[2019-08-09 09:36:40.682] [verbose] CLI downloading, progress=0.8862221070780563
[2019-08-09 09:36:40.684] [verbose] CLI downloading, progress=0.8862221070780563
[2019-08-09 09:36:41.183] [verbose] CLI downloading, progress=0.8973460370683813
[2019-08-09 09:36:41.184] [verbose] CLI downloading, progress=0.8973460370683813
[2019-08-09 09:36:41.690] [verbose] CLI downloading, progress=0.9084653938970156
[2019-08-09 09:36:41.691] [verbose] CLI downloading, progress=0.9084653938970156
[2019-08-09 09:36:42.184] [verbose] CLI downloading, progress=0.9104183244874016
[2019-08-09 09:36:42.186] [verbose] CLI downloading, progress=0.9104183244874016
[2019-08-09 09:36:42.685] [verbose] CLI downloading, progress=0.9104183244874016
[2019-08-09 09:36:42.687] [verbose] CLI downloading, progress=0.9104183244874016
[2019-08-09 09:36:43.185] [verbose] CLI downloading, progress=0.9104183244874016
[2019-08-09 09:36:43.188] [verbose] CLI downloading, progress=0.9104183244874016
[2019-08-09 09:36:43.686] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:43.687] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:44.187] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:44.188] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:44.689] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:44.690] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:45.189] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:45.190] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:45.690] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:45.692] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:46.193] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:46.196] [verbose] CLI downloading, progress=0.9152983643825209
[2019-08-09 09:36:46.694] [verbose] CLI downloading, progress=0.9158828716111294
[2019-08-09 09:36:46.696] [verbose] CLI downloading, progress=0.9158828716111294
[2019-08-09 09:36:47.201] [verbose] CLI downloading, progress=0.916467378839738
[2019-08-09 09:36:47.203] [verbose] CLI downloading, progress=0.916467378839738
[2019-08-09 09:36:47.700] [verbose] CLI downloading, progress=0.9176363932969549
[2019-08-09 09:36:47.701] [verbose] CLI downloading, progress=0.9176363932969549
[2019-08-09 09:36:48.200] [verbose] CLI downloading, progress=0.9182254736872543
[2019-08-09 09:36:48.202] [verbose] CLI downloading, progress=0.9182254736872543
[2019-08-09 09:36:48.701] [verbose] CLI downloading, progress=0.9193944881444713
[2019-08-09 09:36:48.702] [verbose] CLI downloading, progress=0.9193944881444713
[2019-08-09 09:36:49.202] [verbose] CLI downloading, progress=0.9205635026016884
[2019-08-09 09:36:49.203] [verbose] CLI downloading, progress=0.9205635026016884
[2019-08-09 09:36:49.702] [verbose] CLI downloading, progress=0.9217370902205962
[2019-08-09 09:36:49.703] [verbose] CLI downloading, progress=0.9217370902205962
[2019-08-09 09:36:50.202] [verbose] CLI downloading, progress=0.9229061046778133
[2019-08-09 09:36:50.204] [verbose] CLI downloading, progress=0.9229061046778133
[2019-08-09 09:36:50.702] [verbose] CLI downloading, progress=0.9246641995253296
[2019-08-09 09:36:50.704] [verbose] CLI downloading, progress=0.9246641995253296
[2019-08-09 09:36:51.203] [verbose] CLI downloading, progress=0.9271970641826333
[2019-08-09 09:36:51.205] [verbose] CLI downloading, progress=0.9271970641826333
[2019-08-09 09:36:51.703] [verbose] CLI downloading, progress=0.930713253877666
[2019-08-09 09:36:51.704] [verbose] CLI downloading, progress=0.930713253877666
[2019-08-09 09:36:52.205] [verbose] CLI downloading, progress=0.9348093776396165
[2019-08-09 09:36:52.206] [verbose] CLI downloading, progress=0.9348093776396165
[2019-08-09 09:36:52.705] [verbose] CLI downloading, progress=0.9377364869443499
[2019-08-09 09:36:52.706] [verbose] CLI downloading, progress=0.9377364869443499
[2019-08-09 09:36:53.206] [verbose] CLI downloading, progress=0.9435907055538166
[2019-08-09 09:36:53.207] [verbose] CLI downloading, progress=0.9435907055538166
[2019-08-09 09:36:53.714] [verbose] CLI downloading, progress=0.9517829530777175
[2019-08-09 09:36:53.715] [verbose] CLI downloading, progress=0.9517829530777175
[2019-08-09 09:36:54.215] [verbose] CLI downloading, progress=0.9601746095061788
[2019-08-09 09:36:54.216] [verbose] CLI downloading, progress=0.9601746095061788
[2019-08-09 09:36:54.716] [verbose] CLI downloading, progress=0.9671978425728627
[2019-08-09 09:36:54.717] [verbose] CLI downloading, progress=0.9671978425728627
[2019-08-09 09:36:55.215] [verbose] CLI downloading, progress=0.9789062797917963
[2019-08-09 09:36:55.217] [verbose] CLI downloading, progress=0.9789062797917963
[2019-08-09 09:36:55.716] [verbose] CLI downloading, progress=0.9935418263154634
[2019-08-09 09:36:55.717] [verbose] CLI downloading, progress=0.9935418263154634
[2019-08-09 09:36:55.940] [verbose] CLI downloading, progress=1
[2019-08-09 09:36:55.941] [verbose] CLI downloading, progress=1
[2019-08-09 09:36:55.943] [verbose] CLI downloading, progress=1
[2019-08-09 09:36:55.944] [verbose] CLI downloading, progress=1
[2019-08-09 09:36:55.946] [verbose] CLI downloading, progress=1
[2019-08-09 09:36:55.948] [verbose] CLI downloading, progress=1
```

Seems like download gets stuck on failed checksum verification.  Upon software restart things seem to behave properly (checksum verification disabled upon just start?).

**Comment by zeroleak** [comment]
hi, thanks for the report, we will investigate this.
It looks like the download ended correctly but was not detected by the GUI. 

can you confirm it was fixed on GUI restart?
checksum is verified on each start/restart, so if it works on restart your checksum is valid ;)

**Comment by meeDamian** [comment]
If I purge `~/Library/Application Support/whirlpool-gui/`, and run `whirlpool-gui.app` it gets stuck every time.

If I let it get stuck, quit, and re-run, it proceeds correctly :).

**Comment by satsonsats** [comment]
Same issue. Same solution. +1

> If I let it get stuck, quit, and re-run, it proceeds correctly :).

**Comment by zeroleak** [comment]
Thanks for the report. 
We couldn't reproduce yet, which OS/version and which JDK do you use?

**Comment by satsonsats** [comment]
Windows 10 and Java 8 update 221

Just hung up for about 5 mins. When closed and reset, it worked.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by zeroleak** [comment]
Should be fixed in whirlpool-gui 0.10.3. Let me know if it's not.
